composer.json

```diff
@@ -29,6 +29,9 @@
         "doctrine/doctrine-bundle": "~1.0",
         "doctrine/orm": ">=2.2,<2.5-dev"
     },
+    "conflict": {
+        "symfony/dependency-injection": "<2.1"
```

Comment:
I added it on conflict because it's not a direct dependency but a dependency of framework-bundle.

Will update it.